flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -58,9 +61,10 @@
 			.noDefaultValue()
 			.withDescription("The reporter class to use for the reporter named <name>.");
 
-	public static final ConfigOption<String> REPORTER_INTERVAL =
+	public static final ConfigOption<Duration> REPORTER_INTERVAL =
```

Comment:
looks like a breaking API change

Reply:
it will be actually backwards compatible as we use `TimeUtils#parseDuration` for parsing.